community/sample-apps/whatsapp-clone-typescript-react/auth-server/README.md

```diff
@@ -0,0 +1,101 @@
+# JWT Authentication server Boilerplate
```

Comment:
Is this auth server an exact duplicate of our existing boilerplate? If there are no modifications, can't we just symlink it?

If there are modifications, is it possible to generalise them?

Reply:
It is not a duplicate of the existing one. Also it contains a file upload API in addition. So it can be more of a custom API server than just Auth too.

Reply:
Can this be a boilerplate with minor modifications?

Reply:
@praveenweb Maybe it's good to use: https://github.com/elitan/hasura-backend-plus it also have file api support.